portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,35 +24,81 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comment data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> comments;
-
-  @Override
-  public void init() {
-    comments = new ArrayList<String>();
-  }
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String userComment = request.getParameter("text-input");
-    if (userComment == null) {
-      userComment = "";
+    if (userComment != null) {
+      storeComment(userComment);
     }
-    comments.add(userComment);
     response.sendRedirect("/index.html");
   }
 
+  /**
+    * Stores user comment as entity in datastore.
```

Comment:
Given that this comment is short, consider this format instead. 
/** Stores user comment as entity in datastore. */
It takes 2 lines less, so it's more compact, and it is supported by the Google style guide (though I can't link it here).

Reply:
I made all of my java comments one line if it was possible to do so!